src/django_aetos/views.py

```diff
@@ -18,12 +19,45 @@ def export_metrics(request):
                 metric["add_help"] = True
             known_metrics.add(metric["name_without_labels"])
             metrics.append(metric)
+    
+    return metrics
+
+def check_auth(request):
+    ip_address = request.META['REMOTE_ADDR']
+    allowed_ips = app_settings.IP_ALLOWLIST
+    
+    if app_settings.ENABLE_IP_ALLOWLIST and ip_address in allowed_ips:
+        return True
+    elif not app_settings.ENABLE_IP_ALLOWLIST:
```

Comment:
probably use `is False` here so we don't interpret `None` (so no value at all) as false.

Reply:
Fixed in 1ddd9c87f8bb9c78f7dc3998e406cf8b562178f4